### PR TITLE
add htmllint test

### PR DESCRIPTION
**Pull request by KrivichaninDenis**
### Description .htmllintrc config:
- _**line-end-style:**_ Line endings must conform to the given style.
- **_indent-width:_** If it is a number and spaces are used for indentation, then spaces used to indent must come in multiples of that number.
- **_indent-style:_** Only spaces may be used for indentation.
- **_doctype-html5:_**  The doctype element must specify html5.
- **_id-class-style:_** A format specifier. By default specifier equals "**-**" .
- **_tag-bans:_** The value of this option is a list of strings, each of which is a tag name. Tags with any of the given names are disallowed.(We use in code tag _`<i></i>`_. This is disalowed tag)

### Add script to run htmllint [test](https://github.com/pub-t/plain-destination/blob/htmllint-test/package.json#L8)

### CODE REVIEW

File: .htmllintrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "line-end-style": "crlf",
```
Comment (irnc) [inline .htmllintrc]: `LF` it should be ;)

File: .htmllintrc [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "indent-width": "2",
+    "indent-style": "spaces",
+     "doctype-html5": true,
```
Comment (irnc) [inline .htmllintrc]: It it funny that indentation is wrong right after `indent-style` ;)

File: index.html [hunk 3]
```diff
@@ -1,81 +1,102 @@
...
-        </p>
-    </div>
+  <div id="out" class="panel">
+    <a href="#form" class="top"><i class="icon-up-big"></i></a>
+    <a href="#map" class="bottom"><i class="icon-down-big"></i></a>
+    <p>
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum
+      sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec,
+      pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec,
```
Comment (irnc) [inline index.html]: Why we need to explicitly wrap lines here? Why they are so long?

**Comment by irnc** [comment]
LGTM, merge after fixes

**Comment by KrivichaninDenis** [comment]
I will fix this errors after meeting

File: index.html (hunk 3 above)
Comment (KrivichaninDenis) [inline index.html]: In this [div](https://github.com/pub-t/plain-destination/pull/20/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR59) was test text

**Comment by KrivichaninDenis** [comment]
Can i merge this branch?

**Pull request merged**